src/Nethermind/Ethereum.Blockchain.Pyspec.Test/PragueTests.cs

```diff
@@ -0,0 +1,24 @@
+// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Ethereum.Test.Base;
+using NUnit.Framework;
+
+namespace Ethereum.Blockchain.Pyspec.Test;
+
+[TestFixture]
+[Parallelizable(ParallelScope.All)]
+[Explicit("These tests are not ready yet")]
```

Comment:
It will be updated after merging all Pectra's PRs: https://github.com/NethermindEth/nethermind/issues/7422